sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -62,6 +62,7 @@
     "lint:terminal": "eslint package.json \"src/**/*.ts\" samples --ext .ts",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
+    "regenerate": "npx autorest swagger/README.md --typescript",
```

Comment:
woah!! Is this all that is needed to regenerate this?

Reply:
In theory, yes!  I would certainly appreciate if you could clone the branch and give it a shot just to make sure.

Reply:
I'm trying it now!

Reply:
It says `No input files provided`. Am I missing something?

Reply:
Forgot to commit the swagger/README.md files, fixed now :/

Reply:
thank you! It works well!